package-lock.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "pros",
-  "version": "0.0.1",
```

Comment:
Shouldn’t have version change

Reply:
uh this is the package-lock.json not the package.json